peps/pep-0735.rst

```diff
@@ -8,7 +8,7 @@ Status: Accepted
 Type: Standards Track
 Topic: Packaging
 Created: 20-Nov-2023
-Post-History: `14-Nov-2023 <https://discuss.python.org/t/29684>`__, `20-Nov-2023 <https://discuss.python.org/t/39233>`__
+Post-History: `14-Nov-2023 <https://discuss.python.org/t/29684>`__, `20-Nov-2023 <https://discuss.python.org/t/39233>`__, `10-Oct-2024 <https://discuss.python.org/t/39233>`__
```

Comment:
Is there a typo? The last two links are identical.

Reply:
I didn't follow the whole discussion but this post might have some context: https://discuss.python.org/t/pep-735-dependency-groups-in-pyproject-toml/39233/328. I think the current procedure should be to add a link to the acceptance post in the Resolution field.

Reply:
(Which is already there, so I don't know why we need this change at all)

Reply:
Yeah, we have two _threads_ in the Post-History, and a direct link to the acceptance _message_ in Resolution. This is correct.

The CI will also fail if you put a message in Post-History.